package.json

```diff
@@ -243,7 +243,7 @@
     "@types/react-dom": "18.2.4"
   },
   "engines": {
-    "node": ">=16.8.0"
+    "node": ">=16.8.0 <20"
   },
   "packageManager": "pnpm@7.24.3"
```

Comment:
You could just bump this to the latest v7 version, and you wouldn't need to restrict the node version to <20; this issue was fixed in 7.32.2+ and 8.3.1+.

Reply:
Oops, you said there were issues and I ignored that, my mistake.

Reply:
That being said, there have been fixes since 7.32.2 so, maybe it works?

Reply:
Agreed bumping PNPM may work - but while it's unclear whether this will break things (awaiting confirmation on what broke last time, given CI didn't catch it), I think this will be useful as an interim measure.